addon/services/docs-search.js

```diff
@@ -72,7 +72,7 @@ export default Service.extend({
 
   loadSearchIndex() {
     if (!this._searchIndex) {
-      this._searchIndex = fetch(this.get('_indexURL'))
+      this._searchIndex = fetch(this.get('_indexURL'), { credentials: 'same-origin' })
```

Comment:
I'm not entirely the right person to ask about these things, but my 2c is that I imagine we would want something like this to be configurable 🤔

I assume this will make your system work and might break other people's (this is an assumption so anyone more knowledgeable please jump in!)

@ebatsi do you think you could make this optional and configurable (somehow) through the environment config? I think that might help it land a bit sooner 👍